checker/manual/fenum-checker.tex

```diff
@@ -115,18 +113,46 @@ \section{Running the Fenum Checker\label{fenum-running}}
 
 \begin{itemize}
   \item
-If you define your own annotation, provide the name of the annotation using the
-\code{-Aquals} option:
+If you define your own annotation(s), provide the name(s) of the annotation(s)
+through the \code{-Aquals} option, using a comma-no-space-separated notation:
 
 \begin{alltt}
-  javac -processor org.checkerframework.checker.fenum.FenumChecker
-        \textit{-Aquals=myproject.qual.MyFenum} MyFile.java ...
+  javac -Xbootclasspath/p:\textit{/full/path/to/myProject/bin}:\textit{/full/path/to/myLibrary/bin} \ttbs
+        -processor org.checkerframework.checker.fenum.FenumChecker \ttbs
+        -Aquals=\textit{myModule.qual.MyFenum} MyFile.java ...
 \end{alltt}
 
+The annotations listed in \code{-Aquals} must be accessible to
+the compiler during compilation in the classpath.  In other words, they must
+already be compiled (and, typically, be on the javac bootclasspath)
+before you run the Fenum Checker with \code{javac}.  It
+is not sufficient to supply their source files on the command line.
+
+You can also provide the fully-qualified paths to a set of directories
+that contain the annotations through the \code{-AqualDirs} option,
+using a colon-no-space-separated notation. For example:
+
+\begin{alltt}
+  javac -Xbootclasspath/p:\textit{/full/path/to/myProject/bin}:\textit{/full/path/to/myLibrary/bin} \ttbs
+        -processor org.checkerframework.checker.fenum.FenumChecker \ttbs
+        -AqualDirs=\textit{/full/path/to/myProject/bin}:\textit{/full/path/to/myLibrary/bin} MyFile.java ...
+\end{alltt}
+
+Note that in these two examples, the compiled class file of the
+\<myModule.qual.MyFenum> annotation must exist in either the \<myProject/bin>
+directory or the \<myLibrary/bin> directory. The following placement of
+the class file will work with the above commands:
+
+\begin{alltt}
+  .../myProject/bin/myModule/qual/MyFenum.class
```

Comment:
Why can't the class file just be in myProject/bin if the -Aquals option is used?